include/envoy/event/dispatcher.h

```diff
@@ -163,6 +164,14 @@ class Dispatcher {
    */
   virtual Event::TimerPtr createTimer(TimerCb cb) PURE;
 
+  /**
+   * Allocates a schedulable callback. @see SchedulableCallback for docs on how to use the wrapped
+   * callback.
+   * @param cb supplies the callback to invoke when the SchedulableCallback is triggered the event
```

Comment:
```suggestion
   * @param cb supplies the callback to invoke when the SchedulableCallback is triggered on the event
```

Reply:
Done.  Wasn't sure if it should have been "on" or "in"